bin/desi_tsnr_afterburner

```diff
@@ -37,7 +41,7 @@ def parse(options=None):
                         help = 'Update pre-existing output summary file (replace or append)')
     parser.add_argument('--details-dir', type=str, default = None, required=False,
                         help = 'Dir. to write per-exposure per-camera files with per-fiber tSNR details')
-    parser.add_argument('--prod', type = str, default = None, required=False,
+    parser.add_argument('--prod', type = str, default = "daily", required=False,
```

Comment:
I also commented on this for another script, but I prefer the previous default of defaulting to $SPECPROD instead of "daily".